Desktop/main.cpp

```diff
@@ -434,10 +434,12 @@ int main(int argc, char *argv[])
 			if(safeGraphics)
 			{
 				std::cout << "Setting special options for software rendering (aka safe graphics)." << std::endl;
-				QCoreApplication::setAttribute(Qt::AA_UseSoftwareOpenGL);
+				//QCoreApplication::setAttribute(Qt::AA_UseSoftwareOpenGL);
+				QCoreApplication::setAttribute(Qt::AA_UseDesktopOpenGL);
+				QQuickWindow::setGraphicsApi(QSGRendererInterface::OpenGL);
```

Comment:
I imagine if we could try different rendering backends by passing startup parameters in order to debug and solve the problem of qtWebengine crashing on certain graphics cards and rendering backends. any ideas? @JorisGoosen

Reply:
Reading https://www.qt.io/blog/new-graphics-integration-in-qt-webengine-6.6-and-even-6.5.1 it looks like you can set it via an environment variable as `QSG_RHI_BACKEND=opengl` and thus do not need the changes you made (for testing anyway).

Not sure if there are other settings that make sense. In any case, we just switched to 6.7 and I guess theyve now removed the old opengl backend. So the new nightly should be behaving differently even in opengl mode if I understand correctly.

With the environment variable it is possible to test if unsetting it makes a difference in 0.18.3 (it was built with 6.5.3 I think)